src/test/java/org/primeframework/mvc/security/BaseUserIdCookieSecurityContextTest.java

```diff
@@ -53,6 +55,12 @@ public class BaseUserIdCookieSecurityContextTest {
 
   @Inject private SessionCookieKeyChanger cookieKeyChanger;
 
+  @Inject
```

Comment:
Can you take a pass through this file, and others and look for static analysis warnings and see if we can resolve them? 

You should see a yellow gutter in IJ. We should get into the habit of reviewing each of these warnings when you are in a file. 

For example, in this part, I see two testes with un-used arguments. We can either remove those arguments, or if you use the IJ intent, you can ignore this warning for this method or test.

Reply:
Done.